cilium/cmd/bpf_policy_get.go

```diff
@@ -37,10 +37,9 @@ var printIDs bool
 
 // bpfPolicyListCmd represents the bpf_policy_list command
 var bpfPolicyListCmd = &cobra.Command{
-	Use:     "list",
-	Aliases: []string{"get"},
```

Comment:
Please update `tests/helpers.bash` to change the invocation of this command, which is ran during the log-gathering scripts.

Reply:
Done.